src/app/loadout-builder/generated-sets/utils.ts

```diff
@@ -211,35 +189,16 @@ export function filterGeneratedSets(
     return true;
   });
 
-  matchedSets = matchedSets.sort(
```

Comment:
Where did sorting move to?

Reply:
Process is keeping them in a sorted order with the insertion sort. We could leave it in to be sure if you like.

Reply:
I pushed it back in. Still not entirely sure if its needed.

Reply:
I think sorting by the stat order is probably important.

Reply:
Yeah it definitely is, I just think it is getting done twice (once in process and once here), just slightly differently.